src/StakingNode.sol

```diff
@@ -123,11 +123,11 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
 
     receive() external payable {
         // Consensus Layer rewards and the validator principal will be sent this way.
-       if (msg.sender != address(stakingNodesManager.delayedWithdrawalRouter())
+        if (msg.sender != address(stakingNodesManager.delegationManager())
```

Comment:
it appears that `msg.sender != address(stakingNodesManager.delegationManager()` isn't required here as the eigen design is to only have withdraws processed with one flow of funds: 

`delegationManager -> eigenPodManager -> eigenPod -> stakingNode`

I don't see and alternative withdraw flow of funds, aside from beacon 'transfers'.

Reply:
made this change

Reply:
looks right